### PR TITLE
removed all examples

### PR DESCRIPTION
**Pull request by nlbahy**

### CODE REVIEW

File: docs/source/get_all_venues.rst [hunk 1]
```diff
@@ -1,23 +0,0 @@
-Getting All Venues
-==========================
-
-::
-
-    >>> import openreview
-    >>> c = openreview.Client(baseurl='https://api.openreview.net')
-    >>> venues = openreview.tools.get_all_venues(c)
```
Comment (melisabok) [inline docs/source/get_all_venues.rst:8]: there is another way of doing this, client.get_group(id='venues'), I'm not sure if the documentation has this.

File: docs/source/get_submission_invitations.rst [hunk 2]
```diff
@@ -1,101 +0,0 @@
-Getting Submissions
-========================================
-
-All the Invitation Ids for Submissions can be retrieved like this::
-
-    >>> import openreview
-    >>> c = openreview.Client(baseurl='https://api.openreview.net')
-    >>> invitations = openreview.tools.get_submission_invitations(c)
```
Comment (melisabok) [inline docs/source/get_submission_invitations.rst:8]: @nlbahy do you mind removing the function `get_submission_invitations` from the tools file?

File: docs/source/get_all_venues.rst (hunk 1 above)
Comment (nlbahy) [reply docs/source/get_all_venues.rst:8]: I did copy this into the docs, I can change it to get_group(id='venues')

File: docs/source/multiprocessing.rst [hunk 3]
```diff
@@ -1,9 +0,0 @@
-Parallel Execution
-==================
-
-The tools package includes utilities that makes it easy to run a set of requests in parallel using a multiprocessing pool:
-
-    >>> notes = list(tools.iterget_notes(client, invitation='ICLR.cc/2019/Conference/-/Blind_Submission'))
-    >>> openreview.tools.parallel_exec(notes, do_work)
```
Comment (melisabok) [inline docs/source/multiprocessing.rst:7]: we could keep this file but change the documentation and mention that `tools.concurrent_get` should do the trick

Comment (melisabok) [reply docs/source/multiprocessing.rst:7]: cc @carlosmondra

File: docs/source/profiles.rst [hunk 4]
```diff
@@ -1,142 +0,0 @@
...
-     'mccallum@cs.umass.edu',
-     'cs.umass.edu']
-
-Conflicts of interest are represented by direct relationships or by common domain affiliations.
-
-
-Add evidence to a profile
-----------------------------
-
-Add DBLP link:
```
Comment (melisabok) [inline docs/source/profiles.rst:73]: do we have these examples in the new doc?

File: docs/source/get_all_venues.rst (hunk 1 above)
Comment (melisabok) [reply docs/source/get_all_venues.rst:8]: venues contains all the venue ids. The host group only contain the root groups.

File: docs/source/profiles.rst (hunk 4 above)
Comment (nlbahy) [reply docs/source/profiles.rst:73]: No, I figured we would not want people to use those examples

File: docs/source/multiprocessing.rst (hunk 3 above)
Comment (carlosmondra) [reply docs/source/multiprocessing.rst:7]: There is no section in the docs for this. If we were to add it, it should go under the Reference section. I would suggest having an openreview-py subsection under Reference. We could also link readthedocs from this subsection.

**Pull request merged**